README.md

```diff
@@ -2,21 +2,47 @@
 
----
+_A foreign language journaling platform for exchanging feedback with native or advanced speakers and building community around shared interests and languages._
 
-Welcome to the Journaly repository!
+## Why?
+
+**We believe that WRITING is perhaps the most undervalued and under-utilized activities for improving foreign language skills.**
+Some major reasons for this are:
+
+- Writing is relatively _effortful_ and practicing it consistently requires building a habit around it.
+- It is hard to find people to give you feedback.
+- Even once finding someone, figuring out how best to give someone feedback isn't obvious and takes a lot of effort.
+- Once you get feedback from soneone, it isn't easy to actually apply, store, organize, and keep track of it over time.
 
-The application is built with JavaScript/TypeScript on both the frontend and backend with a GraphQL API, leveraging powerful frameworks such as React, Next, and Apollo on the frontend and Prisma, Nexus, and Express on the backend, with a PostgreSQL database with Amazon RDS.
+**Journaly is about striving to build excellent software with beautiful, simple, and intuitive User Interface & User Experience Design that solves each of the above problems for our users:**
 
-We use Python for data science, leveraging a number of powerful libraries for data aggregation and analysis, data visualization, and machine learning, such as NumPy, Pandas, Matplotlib, Seaborn, SciKit, and more.
+- We make the writing experience delightful, enjoyable, and easy to engage in; helping users build healthful and sustainable writing habits.
+- We make it incredibly straightfoward to find people to provide users feedback. In fact, users just have to write and the community finds THEM.
+- We build intuitive tools that make it incredibly easy to sweep through someone's post and give them feedback exactly where it belongs.
+- Feedback stays neatly organized and is easy to digest and find again later.
 
 ## Architecture
 
-The data model/DB schema lives in `j-db-client/prisma/schema`, server side code and some utils are in `nexus/`, and front end code constitutes most the rest of the directories. `graphql/` holds the _frontend_ graphql queries.
+Journaly consists of multiple independent services/packages that are all deployed separately.
+We manage the deployment of these services with our own Cloud Infrastructure in Amazon Web Services (AWS).
 
----
+Here is an overview of the project setup:
+
+```
+|-- journaly/
+|---- docs/
+|---- cypress/
+|---- packages/
+|------ web/ # Our core application that users engage with on the web
+|------ j-mail/ # A simple managed queue service for handling transactional emails from within the app
+|------ j-db-client/ # A database client that can be imported into any service to interact with the db
+|-------- datamodel.prisma # our database schema
+|-------- migrations/
+```
+
+More detailed documentation can be found within each directory (WIP).
```

Comment:
@Lanny I went back-and-forth between this simplified version or adding some additional depth below each package. For this first pass I decided to keep it very simple just as a high-level overview, and then directing people to the specific docs for each package if they need more info. Do you have any thoughts?

Reply:
This approach seems reasonable to me. I might even go so far as to say even this breakdown isn't necessary here. Stuff like cypress and docs should be pretty explanatory, and a breakdown of what each package does might be more suitable for packages/readme.md (and we'd have room to expand a little bit on each there)